docs/content/en/docs/use-cases/host-system-changes/linux-kernel-modules.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Monitor Linux Kernel Modules"
+weight: 1
+icon: "overview"
+description: "Monitor Linux Kernel Modules operations"
+---
+
+# Monitoring Kernel Modules operations
+
+Monitoring kernel modules helps to identify processes that load kernel modules to add features,
+to the operating system, to alter host system functionality or even hide their behaviour.
+
+The [monitor-kernel-modules](https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/host-changes/monitor-kernel-modules.yaml) tracing policy can be used to answer the following questions:
+
+> Which process or container is changing the kernel?
```

Comment:
Not clear what this means. Changing the kernel in what manner? I believe that this PR is about modules, and the next 2 bullets cover all aspects of module loading/unloading.

Reply:
It just means we inserted code into kernel, it was changed, this is not the same kernel that was booted.